pandas/core/groupby/grouper.py

```diff
@@ -413,7 +413,6 @@ def _set_grouper(self, obj: NDFrame, sort: bool = False):
         # "NDFrameT", variable has type "None")
         self.obj = obj  # type: ignore[assignment]
         self._gpr_index = ax
-        return self._gpr_index
```

Comment:
This seems okay to me; the result of this method is indeed never used and it looks cleaner to not have a return. cc @jbrockmendel

Reply:
Thanks for the review @rhshadrach! FYI we talked about this a bit at https://github.com/pandas-dev/pandas/pull/46258#discussion_r821265302 if you want a bit more context

Reply:
Marking as resolved, since I'm worried the "changes requested" label might be keeping this out of your queue.

Reply:
I don't think it being resolved or not changes its appearance in e.g. `pandas/pulls` or the GitHub notifications (not sure what else you might mean by queue here).

Reply:
That is what I meant (maybe you had filters for what to review next), but good to know. Thanks!